src/main/java/org/jenkinsci/plugins/registry/notification/events/EventTypePayloadFields.java

```diff
@@ -0,0 +1,11 @@
+package org.jenkinsci.plugins.registry.notification.events;
+
+import net.sf.json.JSONObject;
+
+public interface EventTypePayloadFields {
```

Comment:
Is this used somewhere? interfaces in Jenkins plugins should be handled with extreme care as they are hard to evolve until we really have Java 8's default methods.

Reply:
Deleting (was a leftover from another bad design idea I had at some point)